cmd/demo/slides/201905/sqlflow.slide

```diff
@@ -0,0 +1,53 @@
+SQLFlow
+Brings SQL and AI together.
+21 May 2019
+
+Yang Yang
+Software Engineer, Ant Financial
+yang.y@antfin.com
+
+* Overview
+SQLFlow is a bridge that connects a SQL engine
+
+- MySQL, Alibaba MaxCompute, Hive, ...
+
+with a ML engine
+
+- TensorFlow, XGboost, ...
+
+It extends the SQL syntax to enable model training and inference.
+
+* Syntax: Regular SQL
+.code regular.sql
+
+* Syntax: Train
+.code train.sql
```

Comment:
Just being curious -- can we actually run train.sql from within the Go slide?

Reply:
I don't think so.